src/Miki/AppProps.cs

```diff
@@ -14,8 +14,8 @@ public static class AppProps
 
         public static class Emoji
         {
-            public static string Mekos => "<:mekos:421972155484471296>";
-
+            public static string Mekos 
```

Comment:
These can be const?

Reply:
That is definitely a possibility, but that falls out of the scope of the current commit.